app/controllers/rule_languages_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RuleLanguagesController < ApplicationController
+  before_action :authorize!
+
+  def create
+    create_rule_language
```

Comment:
when issuing the follow create request:

```
curl --location 'https://mobile-content-api-stage.cru.org/tool-groups/1/rules-language' \
--header 'Content-Type: application/vnd.api+json' \
--header 'Accept: application/vnd.api+json' \
--data '{
 "data":   {
   "type": "tool-group-rule-language",
   "attributes": {
    "languages": [
     "en"
    ]
   }
 }
}'
```

I get the following response:
```
{
    "error": {
        "tool_group": [
            "must exist"
        ]
    }
}
```

Reply:
2 things wrong here:
1. it's not picking up the tool-group from the tool-group id in the URL
2. the error format isn't using the jsonapi error format

Reply:
It was solved in last commits from yesterday